rhacs-operator/tasks/main.yml

```diff
@@ -0,0 +1,126 @@
+---
+- name: Testing Red Hat Advanced Cluster Security
+  block:
+    - name: Create stackrox namespace
+      kubernetes.core.k8s:
+        name: stackrox
```

Comment:
I didn't try other namespaces. Does it need to flexible and configurable? The examples shared are pretty simple and straightforward.

Reply:
Never mind. Just changed to target namespaces. It works well.

Reply:
The reason we are using target_namespace is so that the assessments don't need to really worry about creating namespaces and cleaning them up. So, preferably, we'd like to use target_namespace where possible. Thanks for checking.

Reply:
Target namespace it is. Also done!